docs/quickstart.rst

```diff
@@ -138,9 +138,25 @@ We can search for specific entries using :py:meth:`find() <dataset.Table.find>`
    # Get a specific user
    john = table.find_one(name='John Doe')
 
-   # Find by comparison
+   # Find multiple at once
+   winners = table.find(id=[1, 3, 7])
+
+   # Find by comparison operator
+   elderly_users = table.find(age={'>=', 70})
```

Comment:
I presume these should be dictionaries, not sets.

Reply:
i do not see a set? do you mean the shorthand syntax for dicts i'm using?

Reply:
`{'>=', 70}` has a comma in it instead of a colon, which makes it a set. Same one line below.
```
>>> {'>=', 70}
set(['>=', 70])
```

Reply:
oh, maybe its time for me to go sleeping a little bit ^^
you are right, i'll fix that tomorrow